[2025-01-06T07:38:55.624+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.call_dm_fill_data_marts_f scheduled__2025-01-05T00:00:00+00:00 [queued]>
[2025-01-06T07:38:55.633+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.call_dm_fill_data_marts_f scheduled__2025-01-05T00:00:00+00:00 [queued]>
[2025-01-06T07:38:55.633+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-06T07:38:55.781+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): call_dm_fill_data_marts_f> on 2025-01-05 00:00:00+00:00
[2025-01-06T07:38:55.787+0000] {standard_task_runner.py:60} INFO - Started process 225 to run task
[2025-01-06T07:38:55.790+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'call_dm_fill_data_marts_f', 'scheduled__2025-01-05T00:00:00+00:00', '--job-id', '434', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpfnqot_c7']
[2025-01-06T07:38:55.791+0000] {standard_task_runner.py:88} INFO - Job 434: Subtask call_dm_fill_data_marts_f
[2025-01-06T07:38:55.830+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.call_dm_fill_data_marts_f scheduled__2025-01-05T00:00:00+00:00 [running]> on host 403e7aadd7ee
[2025-01-06T07:38:55.905+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='call_dm_fill_data_marts_f' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-05T00:00:00+00:00'
[2025-01-06T07:38:55.906+0000] {sql.py:276} INFO - Executing: CALL dm.dm_fill_data_marts_f('scheduled__2025-01-05T00:00:00+00:00')
[2025-01-06T07:38:55.913+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-06T07:38:55.920+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-06T07:38:55.930+0000] {sql.py:450} INFO - Running statement: CALL dm.dm_fill_data_marts_f('scheduled__2025-01-05T00:00:00+00:00'), parameters: None
[2025-01-06T07:38:56.188+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=call_dm_fill_data_marts_f, execution_date=20250105T000000, start_date=20250106T073855, end_date=20250106T073856
[2025-01-06T07:38:56.244+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-06T07:38:56.264+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
